### PR TITLE
CORTX-33678 : Fix cortx-re Codacy Issues

### PR DESCRIPTION
**Pull request by subhalaxmisa**
Signed-off-by: subhalaxmisa <subhalaxmi.sahoo@seagate.com>

# Problem Statement
- Fix cortx-re Codacy Issues

# Design
-  Fix cortx-re Codacy Issues

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [x] Unit and System Tests are added
- [x] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM
- [x] Jenkins pipelines used for testing

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [x] Side effects on other features (deployment/upgrade)
- [x] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by shailesh-vaidya — COMMENTED** [review]
Please check with @mukul-seagate11 for cla warning

File: jenkins/internal-ci/generic/branch/cortx-ha.groovy [hunk 1]
```diff
@@ -129,8 +129,8 @@ EOF
                 script {
                     def releaseBuild = build job: 'Release', propagate: true
                     env.release_build = releaseBuild.number
-                    env.release_build_location = "http://cortx-storage.colo.seagate.com/releases/cortx/github/$branch/$os_version/" +releaseBuild.number
```
Comment (shailesh-vaidya) [inline jenkins/internal-ci/generic/branch/cortx-ha.groovy:132]: Please test this with one of the component job execution.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: jenkins/internal-ci/generic/branch/cortx-ha.groovy (hunk 1 above)
Comment (subhalaxmisa) [reply jenkins/internal-ci/generic/branch/cortx-ha.groovy:132]: Job : https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/Subha_WS/job/cortx-ha-internal-ci/6/console

File: jenkins/internal-ci/generic/branch/csm-agent.groovy [hunk 2]
```diff
@@ -131,9 +131,9 @@ pipeline {
                                         "* Component Build  :  ${BUILD_NUMBER} \n" +
                                         "* Release Build    :  ${release_build}  \n\n  " +
                                 "h3. Artifact Location  :  \n" +
-                                    "*  " +"${release_build_location} " +"\n\n" +
+                                    "*  " + "${release_build_location} " + "\n\n" +
```
Comment (gauravchaudhari02) [inline jenkins/internal-ci/generic/branch/csm-agent.groovy:134]: remove extra space between double quotes

File: jenkins/internal-ci/generic/branch/csm-agent.groovy [hunk 3]
```diff
@@ -131,9 +131,9 @@ pipeline {
...
                                 "h3. Image Location  :  \n" +
-                                    "*  " +"${cortx_images} " +"\n" +
+                                    "*  " + "${cortx_images} " + "\n" +
```
Comment (gauravchaudhari02) [inline jenkins/internal-ci/generic/branch/csm-agent.groovy:136]: remove extra space between double quotes

File: jenkins/internal-ci/generic/branch/cortx-rgw-integration.groovy [hunk 4]
```diff
@@ -101,9 +101,9 @@ pipeline {
                                         "* Component Build  :  ${BUILD_NUMBER} \n" +
                                         "* Release Build    :  ${release_build}  \n\n  " +
                                 "h3. Artifact Location  :  \n" +
-                                    "*  " +"${release_build_location} " +"\n\n" +
+                                    "*  " + "${release_build_location} " + "\n\n" +
```
Comment (gauravchaudhari02) [inline jenkins/internal-ci/generic/branch/cortx-rgw-integration.groovy:104]: remove extra space between double quotes

File: jenkins/internal-ci/generic/branch/cortx-rgw-integration.groovy [hunk 5]
```diff
@@ -101,9 +101,9 @@ pipeline {
...
                                 "h3. Image Location  :  \n" +
-                                    "*  " +"${cortx_images} " +"\n" +
+                                    "*  " + "${cortx_images} " + "\n" +
```
Comment (gauravchaudhari02) [inline jenkins/internal-ci/generic/branch/cortx-rgw-integration.groovy:106]: remove extra space between double quotes

File: jenkins/internal-ci/generic/branch/cortx-py-utils.groovy [hunk 6]
```diff
@@ -111,9 +111,9 @@ pipeline {
                                             "* Component Build  :  ${BUILD_NUMBER} \n" +
                                             "* Release Build    :  ${release_build}  \n\n  " +
                                     "h3. Artifact Location  :  \n" +
-                                        "*  " +"${release_build_location} " +"\n\n" +
+                                        "*  " + "${release_build_location} " + "\n\n" +
```
Comment (gauravchaudhari02) [inline jenkins/internal-ci/generic/branch/cortx-py-utils.groovy:114]: remove extra space between double quotes

File: jenkins/internal-ci/generic/branch/cortx-py-utils.groovy [hunk 7]
```diff
@@ -111,9 +111,9 @@ pipeline {
...
                                     "h3. Image Location  :  \n" +
-                                        "*  " +"${cortx_images} " +"\n" +
+                                        "*  " + "${cortx_images} " + "\n" +
```
Comment (gauravchaudhari02) [inline jenkins/internal-ci/generic/branch/cortx-py-utils.groovy:116]: remove extra space between double quotes

File: jenkins/internal-ci/generic/branch/cortx-ha.groovy [hunk 8]
```diff
@@ -153,9 +153,9 @@ EOF
                                          "* Component Build  :  ${BUILD_NUMBER} \n" +
                                         "* Release Build    :  ${release_build}  \n\n  " +
                                 "h3. Artifact Location  :  \n" +
-                                    "*  " +"${release_build_location} " +"\n\n" +
+                                    "*  " + "${release_build_location} " + "\n\n" +
```
Comment (gauravchaudhari02) [inline jenkins/internal-ci/generic/branch/cortx-ha.groovy:156]: remove extra space between double quotes

File: jenkins/internal-ci/generic/branch/cortx-ha.groovy [hunk 9]
```diff
@@ -153,9 +153,9 @@ EOF
...
                                 "h3. Image Location  :  \n" +
-                                    "*  " +"${cortx_images} " +"\n" +
+                                    "*  " + "${cortx_images} " + "\n" +
```
Comment (gauravchaudhari02) [inline jenkins/internal-ci/generic/branch/cortx-ha.groovy:158]: remove extra space between double quotes

Comment (subhalaxmisa) [reply jenkins/internal-ci/generic/branch/cortx-ha.groovy]: Done

File: jenkins/internal-ci/generic/branch/cortx-ha.groovy (hunk 8 above)
Comment (subhalaxmisa) [reply jenkins/internal-ci/generic/branch/cortx-ha.groovy]: Done

File: jenkins/internal-ci/generic/branch/cortx-py-utils.groovy (hunk 7 above)
Comment (subhalaxmisa) [reply jenkins/internal-ci/generic/branch/cortx-py-utils.groovy]: Done

File: jenkins/internal-ci/generic/branch/cortx-py-utils.groovy (hunk 6 above)
Comment (subhalaxmisa) [reply jenkins/internal-ci/generic/branch/cortx-py-utils.groovy]: Done

File: jenkins/internal-ci/generic/branch/cortx-rgw-integration.groovy (hunk 5 above)
Comment (subhalaxmisa) [reply jenkins/internal-ci/generic/branch/cortx-rgw-integration.groovy]: Done

File: jenkins/internal-ci/generic/branch/cortx-rgw-integration.groovy (hunk 4 above)
Comment (subhalaxmisa) [reply jenkins/internal-ci/generic/branch/cortx-rgw-integration.groovy]: Done

File: jenkins/internal-ci/generic/branch/csm-agent.groovy (hunk 3 above)
Comment (subhalaxmisa) [reply jenkins/internal-ci/generic/branch/csm-agent.groovy]: Done

File: jenkins/internal-ci/generic/branch/csm-agent.groovy (hunk 2 above)
Comment (subhalaxmisa) [reply jenkins/internal-ci/generic/branch/csm-agent.groovy]: Done

**Review by shailesh-vaidya — APPROVED** [review]

**Review by gauravchaudhari02 — APPROVED** [review]

**Pull request merged**